types/src/keyless/openid_sig.rs

```diff
@@ -113,15 +112,15 @@ impl OpenIdSig {
         Ok(())
     }
 
-    pub fn verify_jwt_signature(
-        &self,
-        rsa_jwk: &RSA_JWK,
-        jwt_header_b64: &String,
-    ) -> anyhow::Result<()> {
-        let jwt_payload_b64 = &self.jwt_payload_b64;
-        let jwt_sig_b64 = &self.jwt_sig_b64;
-        let jwt_token = format!("{}.{}.{}", jwt_header_b64, jwt_payload_b64, jwt_sig_b64);
-        rsa_jwk.verify_signature(&jwt_token)?;
+    /// `jwt_header` is the *decoded* JWT header (i.e., *not* base64url-encoded)
+    pub fn verify_jwt_signature(&self, rsa_jwk: &RSA_JWK, jwt_header: &str) -> anyhow::Result<()> {
```

Comment:
call jwt_header_json?

Reply:
will do it later
need this finish CI/CD & merge

Reply:
Fixed now.